scripts/west_commands/fetchers/fetcher.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from abc import ABC, abstractmethod
+from collections import namedtuple
+
+FetcherMeta = namedtuple('FetcherMeta', ['schemes', 'module'])
+
+fetchersmeta = [
+
+        # List your fetcher here, implementation must be placed in
```

Comment:
Might want to include a request to preserve alphabetical order before this gets out of hand

Reply:
I have rewritten this bit instead